CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` when updating to this version!
 
+- Minor: Allow configuring streamer, bot, admin, and control hub using Twitch User IDs instead of Twitch User Logins. (#1590)
```

Comment:
```suggestion
- Minor: Allow configuring streamer, bot, admin, and control hub using Twitch User IDs instead of Twitch User Logins. **Specifying user logins in the config has therefore now been deprecated and may be stop working in a future release.** (#1590)
```

Reply:
I'm not sure if I agree with this. I mean we haven't removed the password in config file yet have we? We still just spit a deprecation warning.

Reply:
Well, that's what deprecated means - still works but we plan on removing it in the future.